integration_test/geth_blockchain_test.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/vulcanize/vulcanizedb/test_config"
 )
 
+func init() {
+	//log.SetOutput(ioutil.Discard)
```

Comment:
Assuming this can either be deleted or uncommented

Reply:
deleted